test/onigumo_test.exs

```diff
@@ -9,6 +9,22 @@ defmodule OnigumoTest do
 
   setup(:verify_on_exit!)
 
+  @tag :tmp_dir
+  test("run the download", %{tmp_dir: tmp_dir}) do
+    expect(HTTPoisonMock, :start, fn -> nil end)
```

Comment:
This assertion is mandatory. Mox requires all calls to have an _expect_ clause.

Reply:
Line 14 is necessary I've tested...